.github/workflows/tests.yml

```diff
@@ -35,12 +35,13 @@ jobs:
       - name: Setup Bundler
         run: |
           bundle config path ~/vendor/bundle
+          bundle install
 
       - name: Run ShellCheck
-        run: rake test:shellcheck
+        run: bundle exec rake test:shellcheck
 
       - name: Run unit tests
-        run: rake test:unit
+        run: bundle exec rake test:unit
```

Comment:
There's something wrong with the CI here. unit tests keep installing gems every time when they should have them cached.